### DISCUSSION TITLE
Electronic Frontier Foundation advice

### DISCUSSION THREAD

**Discussion by alansill** [hpc-social/mastodon-instance-and-policies#7] [General]
Ran into [this article](https://www.eff.org/deeplinks/2022/12/user-generated-content-and-fediverse-legal-primer) from [EFF](https://www.eff.org/) entitled "User Generated Content and the Fediverse: A Legal Primer" that contains some useful current information for reference for Fediverse service providers. Note that our policies as in this repository are in keeping with these guidelines at the moment, so I'm just parking this link here for reference.

**Comment by vsoch** [comment]
Reported and blocked. Apologies to all the usernames that got linked above.
